packages/yoastseo/src/parse/traverse/innerText.js

```diff
@@ -14,6 +14,8 @@ export default function innerText( node ) {
 		node.childNodes.forEach( child => {
 			if ( child.name === "#text" ) {
 				text += child.value;
+			} else if ( child.name === "br" ) {
+				text += " ";
```

Comment:
Adding a space for the `br` tag means that we're adding one more character to the text length. This will result in inaccurate position information of the tokens following the `br` element